rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -18,55 +18,42 @@
 
 
 def compile_wrapper(rl_module: "TorchRLModule", compile_config: TorchCompileConfig):
-    """A wrapper that compiles the forward methods of a TorchRLModule.
-
-    Possibly replaces the forward methods with compiled versions, depending on
-    compile_config. Note that this also makes it so that exploration and inference
-    do not trace grads because this can lead to memory leaks and slowness.
-    """
+    """A wrapper that compiles the forward methods of a TorchRLModule."""
 
     # TODO(Artur): Remove this once our requirements enforce torch >= 2.0.0
     # Check if torch framework supports torch.compile.
     if (
         torch is not None
         and version.parse(torch.__version__) < TORCH_COMPILE_REQUIRED_VERSION
-        and (
-            compile_config.compile_forward_train
-            or compile_config.compile_forward_inference
-            or compile_config.compile_forward_exploration
-        )
```

Comment:
We can just compile all the forward methods. Ones that are not called, will not be traced anyway.